### ISSUE TITLE
BUG: 

### ISSUE THREAD

**Issue by BananenRitter** [zsviczian/obsidian-excalidraw-plugin#1207]
SYSTEM INFO:
	Obsidian version: v1.3.5
	Installer version: v1.3.4
	Operating system: Windows 10 Home 10.0.19045
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Everforest v0.0.0
	Snippets enabled: 3
	Restricted mode: off
	Plugins installed: 68
	Plugins enabled: 29
		1: Excalidraw v1.7.25
		2: Advanced Tables v0.17.3
		3: Mousewheel Image zoom v1.0.17
		4: Highlightr v1.2.2
		5: cMenu v1.1.2
		6: Admonition v9.2.3
		7: AutoMOC v1.1.1
		8: Buttons v0.4.19
		9: Google Calendar v1.9.12
		10: Calendar v1.5.10
		11: Surfing v0.8.16
		12: Citations v0.4.5
		13: Copy button for code blocks v0.1.0
		14: Dataview v0.5.55
		15: Dictionary v2.22.0
		16: Editor Syntax Highlight v0.1.3
		17: Excel to Markdown Table v0.4.0
		18: Find orphaned files and broken links v1.9.0
		19: Footnote Shortcut v0.1.0
		20: Image Toolkit v1.3.1
		21: Kanban v1.5.2
		22: Markdown Formatting Assistant v0.4.1
		23: Outliner v4.6.0
		24: Pandoc Plugin v0.4.1
		25: Recent Files v1.3.5
		26: Tag Wrangler v0.5.8
		27: Zotero Integration v2.3.8
		28: Latex Suite v1.7.2
		29: Diagrams v1.5.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

Please run `Command Palette/Show Debug info` in Obsidian and paste the result here.

1. live preview; Excalidraw Draw.io; diagram and diagram link are invisible after creating in note even if the ! is set before the diagram embed link; TEMPORARY SOLUTION: find the invisible link and put the cursor on the right place to see it, delete the !, activate read or source mode, go back and set the ! again

2. cant see edit option when right clicking on diagram picture

3. diagram isnt opened in edit mode when opening link with right click (not important if 2. works)

4. Settings in draw.io arent globally saved. I always have to click on more shapes to add all the ones i want to get displayed, style and diagram settings in the right menu also arent saved globally (im not sure if you even have influence on this since another plugin is required for diagrams to work)

Right click
5. Click on '....'
6. Scroll down to '....'
7. See error

1. diagram and link text should be always displayed

2. right clicking on a diagram picture should always contain the edit diagram option

3. open diagram should always open edit mode (in my opinion) (not important if 2. works)

4. all settings should be saved globally or at least there should be an option to either save options locally for each diagram or globally for all, same goes for more shapes to display all the tabs you want

I cant show invisible things and it wouldnt change anything to do that

**Comment by zsviczian** [comment]
If you embed a draw.io SVG to Excalidraw you need to open excalidraw and click the draw.io embedded image to edit it. You cannot directly edit the draw.io via Excalidraw from live preview.

You can however create a separate draw.io drawing using the features of the diagrams plugin, and embed the resulting SVG into Excalidraw separately.

you may also modify the draw.io script in the script library to insert a link along side the draw.io image into your Excalidraw drawing.

**Comment by zsviczian** [comment]
for further clarity - the draw.io plugin is not one that I maintain. All I did was to create a way to embed draw.io diagrams into Excalidraw using the other plugin and to allow them to be opened directly from Excalidraw. Any issues that are draw.io specific, please consult the developer for the other plugin.,

**Comment by BananenRitter** [comment]
oops i made a mistake. Yes these bugs arent related to your plugin. Unfortunately the draw.io plugin is not maintained since 2 years :(

**Comment by zsviczian** [comment]
yes, currently the draw.io plugin only provides a super basic level of integration.
